src/foam/nanos/controller/ApplicationController.js

```diff
@@ -168,6 +168,10 @@ foam.CLASS({
         self.loginSuccess = !! result;
         if ( result ) {
           self.user.copyFrom(result);
+          if ( ! self.user.emailVerified ) {
+            self.stack.push({ class: 'net.nanopay.auth.email.ResendVerificationEmail' });
```

Comment:
Hmm so you're referencing a class in our repo. So this view may actually need to be moved here. If you move the view. I can make the modifications to move the email verification service to foam2.